### DISCUSSION TITLE
offline mode

### DISCUSSION THREAD

**Discussion by A1268** [bromite/bromite#1727] [Q&A]
how do you disable offline mode? 

i don't like how if i run into a bad connection the browser will download  a site for offline viewing when connection is back and i get a tick mark telling page is ready to view.

how do i stop this? can not find option in flags

thank you

**Comment by uazo** [comment]
there is an internal flag in chromium `kOfflinePagesNetworkStateLikelyUnknown` which is not accessible from the configuration, but which seems to disable the offline function.
it would probably have to be made manageable.

**Reply by jakariyaa** [reply]
Is it been possible to do so? I have root access and I use custom rom. This offline mode things really bugs me a lot. Any help would be highly appreciated 👍

**Comment by csagan5** [comment]
It seems related to how tabs are serialised; the flag @uazo mentioned would be a good way to expose to the user.
